### DISCUSSION TITLE
Chinese language

### DISCUSSION THREAD

**Discussion by 8838** [isontheline/pro.webssh.net#1143] [Ideas]
There is no Chinese language in the webssh settings. I hope you can improve it.

**Comment by 8838** [comment]
![](https://i.imgur.com/zrsnrqM.png)

**Comment by isontheline** [comment]
Already available on the top options of the settings :
<img width="327" alt="image" src="https://github.com/isontheline/pro.webssh.net/assets/44212923/d72ad8c1-e7bc-4064-a8bf-c1e04ded51a2">

**Comment by 8838** [comment]
No such option

![IMG_1347](https://github.com/isontheline/pro.webssh.net/assets/44350798/2f369b05-2213-40e6-ba5f-e09df8691d59)

**Comment by isontheline** [comment]
Please do the following steps in order to debug :
1. Exit WebSSH (kill)
2. Launch WebSSH
3. Go to WebSSH Settings (Gear)
4. Go to Logs
5. Show all logs

Please send me strings like these :
```
[WebSSH] [DBG] Device language : en
[WebSSH] [DBG] Language bundle is not loaded. Will load this bundle : en
```

**Comment by 8838** [comment]
Device language : zh-Hans
Language bundle is not loaded. Will load this bundle : zh-Hans

I think I found the problem. In iOS, the Simplified Chinese identification symbol is not "zh-hans", but "zh-cn". I don't know if this is correct, but I did encounter this problem in Safari before. Just for reference

**Reply by isontheline** [reply]
So WebSSH appears in English instead?

**Comment by 8838** [comment]
If ios cannot recognize "zh-hans", it will appear in English. This is just a guess, for reference only. I believe you will find the answer

**Comment by 8838** [comment]
Just to clarify, the language button you marked in your screenshot only changes the language within the app. I'm talking about the language in the settings, not the language within the app.

**Comment by 8838** [comment]
I mean the language here (↓)

![IMG_1348](https://github.com/isontheline/pro.webssh.net/assets/44350798/0e871842-ea56-4381-a575-f1ebe6c6aa90)

**Reply by isontheline** [reply]
OK 😅 I know understand
Settings are not translated in any language
I need to make some work on it
